recipes/nss/all/conandata.yml

```diff
@@ -1,40 +1,16 @@
 sources:
+  "3.107":
+    url: "https://ftp.mozilla.org/pub/security/nss/releases/NSS_3_107_RTM/src/nss-3.107.tar.gz"
+    sha256: "7f7e96473e38150771a615f5d40e8c41ba3a19385301ae0c525091f2fc9d6729"
   "3.93":
     url: "https://ftp.mozilla.org/pub/security/nss/releases/NSS_3_93_RTM/src/nss-3.93.tar.gz"
     sha256: "15f54bb72048eb105f8c0e936a04b899e74c3db9a19bbc1e00acee2af9476a8a"
-  "3.92":
```

Comment:
Why are you removing these ?

We are migrating from debian based builds to conan based which currently brings 3.87.1 (so that version would even be missing for us) and in order to minimize risk, we try as much as we can to use the same lib version.

Was there any problem with this or it was solely to minimize the number of patches to make for all those versions ?

Reply:
> was solely to minimize the number of patches to make for all those versions ?

Yes. This was an extremely difficult recipe to get right (i.e. to add support on all platforms and cross-compilation support) to due to its very uncommon build system, so I preferred to keeps simple. I can restore any specific versions you might need.

Reply:
Since you already apply the patch to 2 versions (3.107 and 3.93), can't it be applied to all of them ? I mean the worse case was to find out what to change, are those build files changing across version ?

We'd need at least version `3.87.1` (which wasn't in the original list) to help us with our migration to conan based build (before we attempt to upgrade to a later version of nss)